tsconfig.json

```diff
@@ -1,34 +1,30 @@
 {
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig.json to read more about this file */
```

Comment:
The transpilation for the reporter requires this to be a strictly formatted json file, so I've removed the comments here.